packages/vue/src/components/field/field-input.vue

```diff
@@ -8,17 +8,28 @@ export interface FieldInputProps
     /**
      * @vue-ignore
      */
-    InputHTMLAttributes {}
+    Omit<InputHTMLAttributes, 'value'> {
+  modelValue?: InputHTMLAttributes['value']
```

Comment:
Have you looked into the helper `defineModel()` https://vuejs.org/guide/components/v-model.html?

I do think this removes the need to add the prop and emit in this case. But I'm currently not really aware if the emit for example is needed specifically somewhere.

Reply:
I tried it and it wasn't yielding the expected results. Figured to take the old approach for now and reevaluate the issue later.

Reply:
I tried applying it again, and the value just does not want to emit. Maybe need to take a look at the factory

Reply:
Yep, something is happening in the factory to prevent `defineModel` from functioning at the component level

Reply:
Alright, let's keep it this way for now and we can improve it in the future